doc/ref_impl/cntt-ri/chapters/chapter06.md

```diff
@@ -61,241 +63,400 @@ Thanks to the descriptor file, the NFVi infrastructure deployment could be compl
-| network_card_bond_infos | List | No | NIC bonding, might not be always the case.|
+| IP_TYPE | String | Yes | IPV4 or IPV6 |
+| NETWORK_TYPE | String | Yes | VLAN or VXLAN  |
+| TIMEZONE | String | Yes | The timezone where VIM deployed, like UTC+8  |
+| STORAGE_TYPE | String | Yes |  The Storage type, for example "ceph" |
+| HUGEPAGE_ENABLE | String | Yes | TRUE or FALSE  |
+| HUGEPAGE_SIZE | String | Yes | The storage size that hypervisor set for VM, for example "1GB" |
+| QINQ_ENABLE | String | Yes | TRUE or FALSE |
+| HYPERVISOR_CORES | String | Yes | number of CPU bounded for Hypervisor |
+| EXTERNAL_NTP_SERVER_IP | List | Yes | IP list of NTP server, seperated by comma, for example: primariy_IP;second_IP |
 
-<p align="center"><b>Table 6-3-2-1:</b> Server template.</p>
+<p align="center"><b>Table 6-3-2:</b> Global Settings </p>
 
-This is the network interface definition. Generally, there are a list of interfaces included in server template.
+
+<a name="6.3.3"></a>
+### 6.3.3 Parameters for network virtualization
+MTU value for network virtualization should be defined, this is usually standard value that defined by user.
+
+3 instances are expected for Manage Service and storage,  they may have different MTU requirement.
 
 | Field # | type | mandatory | Instruction |
 |----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| port_name | String | Yes | port name, e.g ens0,ens1 |
+| NIC_PORT_TYPE | String | Yes | Type for nic port, Manage, Service or Storage |
+| MTU | String | Yes | This describes the row where rack located |
 
-<p align="center"><b>Table 6-3-2-2:</b> Network Card Bond information.</p>
+<p align="center"><b>Table 6-3-3:</b> Network Virtualization parameter .</p>
 
-In some case, interface would be bonded together. This is the network interface bond definition.
 
-| Field # | type | mandatory | Instruction |
-|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| bond_name | String | Yes | new bonded name. e.g bond 0 is the new name after bond eno0,eno1.|
-| bond_type | String | Yes | dvs,sr-iov,ovs |
-| bond_mode | String | Yes | balance-tcp,balance-slb,balance-backup |
-| bond_members | String | Yes | members of interface name to bond in together, e.g eno0,eno1. |
+<a name="6.3.4"></a>
+### 6.3.4 server information
+Server information should be provided for installer, including full detail info. for each server, nic mapping etc.
 
-<p align="center"><b>Table 6-3-2-3:</b> Network Card Bond information.</p>
+#### 6.3.4.1 server information
+First, a table describes the information for each server in the resource pool should be provided.
```

Comment:
A table describing information for each server in the resource pool shall be provided

Reply:
Thanks!